packages/identifier/src/reserved.ts

```diff
@@ -1070,6 +1070,8 @@ const slurs = [
   'heeb',
   'heebs',
```

Comment:
I'm not an expert, but from quick research we should be probably be censoring "hebe" and not "heeb," since the latter seems to be part of a reclamation effort?